### PR TITLE
Added local database support, will add SQL queries later

### PR DESCRIPTION
**Pull request by FaLxy**
To add the SQL queries that you want to run when the Local Database is spun up, add your queries to Database_schema.sql

### CODE REVIEW

File: Database_schema.sql
Comment (GetPsyched) [inline Database_schema.sql:1]: move to `database/schema.sql`

File: Dockerfile_database
Comment (GetPsyched) [inline Dockerfile_database:1]: move to `database/Dockerfile`

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    build:
+      context: ./
+      dockerfile: Dockerfile_database
+    container_name: 'isac'
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_USER=ISAC_ADMIN
+      - POSTGRES_PASSWORD=pass
+      - POSTGRES_DB=ISAC
```
Comment (Maan2003) [inline docker-compose.yaml:14]: please add new lines in your files
-- we need a pre-commit check for this

Comment (FaLxy) [reply docker-compose.yaml:14]: new lines in which file?

Comment (Maan2003) [reply docker-compose.yaml:14]: in this file, see the error
image

Comment (FaLxy) [reply docker-compose.yaml:14]: nvm got it

Comment (FaLxy) [reply docker-compose.yaml:14]: is this error? Or a convention?

Comment (Maan2003) [reply docker-compose.yaml]: yes it is convention for text based files, file should end with a new line

**Review by GetPsyched — APPROVED** [review]

**Pull request merged**